build-tools/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -113,7 +113,8 @@ public class ElasticsearchNode implements TestClusterConfiguration {
     private static final List<String> OVERRIDABLE_SETTINGS = Arrays.asList(
         "path.repo",
         "discovery.seed_providers",
-        "cluster.deprecation_indexing.enabled"
+        "cluster.deprecation_indexing.enabled",
+        "cluster.initial_master_nodes"
```

Comment:
this is here since if we change the master node configuration we need to update the cluster.initial_master_nodes

Reply:
Is this something we intend folks to do? How is it that none of our existing tests require such a setting? What scenario requires setting this setting that is not required by any of our existing test suites?

Reply:
> Is this something we intend folks to do?

For this specific use case of manually debugging - yes. It allows you to _not_ have the master node role on every node. 

> How is it that none of our existing tests require such a setting?

I believe most tests that require fine control over node roles (specifically which node is master) is handled via other types of tests. 

> What scenario requires setting this setting that is not required by any of our existing test suites?

Starting a new cluster with multiple nodes where one of the nodes is not a master node.

Reply:
...actually i did find one example, where I think it should require this (but didn't)  --> x-pack/qa/mixed-tier-cluster/build.gradle . I will try to track down how that is working, maybe i am missing something.

Reply:
OK.. I think I figured out why that case worked (./gradlew :x-pack:qa:mixed-tier-cluster:bwcTest)  and my manual testing hung when trying to start.  In my example, I have 2 nodes (1 master) but in the example there are 3 nodes (2 master). So with 2/3 masters it can form a quorum, but not with 1/2. If I manually test with 2/3 masters i do not need to override `cluster.initial_master_nodes`.  When it hangs it is pretty hard to understand why and killing the gradle task leaves the ES processes running which then causes more hard to diagnose failures since the process occupies the ports.  I would prefer to leave this as-is if possible and keep it's usage in the example to avoid having to troubleshoot this scenario.